web/index.php

```diff
@@ -30,9 +30,14 @@
 // Instantiate a Slim app
 $app = new Slim(array(
     'log.enabled' => true,
-    'templates.path' => '../templates'
+    'templates.path' => __DIR__.'/../templates'
```

Comment:
I'm wondering if I had to do this because I didn't set the `SLIM_MODE` property when deploying to Heroku (because I wasn't using the Heroku button to deploy).